app/src/main/java/org/simple/clinic/recentpatient/RecentPatientRelativeTimeStampGenerator.kt

```diff
@@ -0,0 +1,50 @@
+package org.simple.clinic.recentpatient
+
+import android.content.Context
+import org.simple.clinic.R
+import org.threeten.bp.LocalDate
+import org.threeten.bp.ZoneOffset
+import org.threeten.bp.format.DateTimeFormatter
+import java.util.Locale
+import javax.inject.Inject
+
+class RecentPatientRelativeTimeStampGenerator @Inject constructor() {
```

Comment:
We have both unit and instrumented tests for the `RelativeTimestampGenerator` which is being used in Patient Summary. Can you add similar tests for this class as well?

Reply:
As discussed privately, we'll move the presentation logic to the view item types. So, we'll not require instrumented tests.